lib/transforms/push.js

```diff
@@ -75,81 +79,97 @@ async function enqueuePush(payload, jiraHost) {
  */
 function processPush(app) {
   return async function (job) {
-    const {
-      repository,
-      repository: { owner, name: repo },
-      shas,
-      installationId,
-      jiraHost,
-    } = job.data;
-
-    const subscription = await Subscription.getSingleInstallation(
-      jiraHost,
-      installationId,
-    );
-
-    if (!subscription) return {};
-
-    const jiraClient = await getJiraClient(subscription.jiraHost, installationId, app.log);
-    const github = await app.auth(installationId);
-    enhanceOctokit(github, app.log);
-
-    const commits = await Promise.all(
-      shas.map(async sha => {
-        const {
-          data,
-          data: { commit: githubCommit },
-        } = await github.repos.getCommit({ owner: owner.login, repo, ref: sha.id });
-        const { files } = data;
-        // Not all commits have a github author, so create username only if author exists
-        const username = data.author ? data.author.login : undefined;
-        // Jira only accepts a max of 10 files for each commit, so don't send all of them
-        const filesToSend = files.slice(0, 10);
-        return {
-          hash: data.sha,
-          message: githubCommit.message,
-          author: {
-            avatar: username ? `https://github.com/${username}.png` : undefined,
-            email: githubCommit.author.email,
-            name: githubCommit.author.name,
-            url: username ? `https://github.com/${username}` : undefined,
-          },
-          authorTimestamp: githubCommit.author.date,
-          displayId: data.sha.substring(0, 6),
-          fileCount: files.length, // Send the total count for all files
-          files: filesToSend.map(mapFile),
-          id: data.sha,
-          issueKeys: sha.issueKeys,
-          url: data.html_url,
+    try {
```

Comment:
What's the reason for the try/catch? did we have an issue with one of the calls?

Reply:
Just a precautionary measure given the new change. Think I should remove it?

Reply:
No,  I think it's good considering what happened with the last publish, I was just curious if there was an explicit reason for it that we needed to bring up with github.